### PR TITLE
add the Spanish translation for distribute node apps, part1

### PR DESCRIPTION
**Pull request by come-maiz**
Notes for reviewers:

* [Video](https://www.youtube.com/watch?v=S3YvRALc2C0)
* [English description](https://github.com/elopio/snapcraft-videos/blob/master/distribute_node_apps/part1/description.txt)
* [English subtitles](https://github.com/elopio/snapcraft-videos/blob/master/distribute_node_apps/part1/subtitles.srt)
* Please leave your review comments in English.

### CODE REVIEW

File: distribute_node_apps/part1/subtitles-es.txt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+00:00:39,500 --> 00:00:43,059
+Si hubiese registrado un nombre distinto antes, asegúrese de usarlo aquí.
+
+8
+00:00:47,800 --> 00:00:51,079
+Ahora, agregue una parte para describir cómo se construye su aplicación.
+
+9
+00:00:53,730 --> 00:00:58,790
+Use el plugin de nodejs de snapcraft, que sabe cómo manejer aplicaciones nodejs.
```
Comment (allanesqui) [inline distribute_node_apps/part1/subtitles-es.txt]: we use the node.js > nosotros usamos el plugin de node.js de snapcraft

File: distribute_node_apps/part1/subtitles-es.txt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+10
+00:01:02,120 --> 00:01:07,330
+Empezaremos con devmode, que significa que la aplicación no está confinada. Podemos asegurarla luego.
```
Comment (allanesqui) [inline distribute_node_apps/part1/subtitles-es.txt]: devmode in the original text is in capital letter> Devmode

**Review by allanesqui — COMMENTED** [review]
Add some translation suggestions

File: distribute_node_apps/part1/subtitles-es.txt (hunk 2 above)
Comment (come-maiz) [reply distribute_node_apps/part1/subtitles-es.txt]: I will instead update the original.

File: distribute_node_apps/part1/subtitles-es.txt (hunk 1 above)
Comment (come-maiz) [reply distribute_node_apps/part1/subtitles-es.txt]: I've updated it to "Usamos el plugin..."

**Comment by come-maiz** [comment]
Thanks a lot for the review @allanesqui!
I've pushed an update. Please check it and if you are happy, mark the pr as approved.

**Pull request merged**